Add git slides

maxnoe opened the pull request.

maxnoe commented:
@vuillaut Ready

vuillaut commented:
When your content is ready, please also updates the pages:
`docs/content/posts/clase04.md`
Thanks

maxnoe commented:
@vuillaut I updated the git post 1 and removed the second, since it's only one set of slides and really a single lecture with a break. Hope that's ok.
